diesel/src/macros/mod.rs

```diff
@@ -194,6 +198,26 @@ macro_rules! __diesel_column {
 /// # fn main() {}
 /// ```
 ///
+///
+/// If you want to add documentation for the generated code you could use the
+/// following syntax:
+///
+/// ```
+/// #[macro_use] extern crate diesel;
+///
+/// table! {
+///
+///     /// Table documentation
+///     posts {
+///         /// Id column documentation
+///         id -> Integer,
+///         /// Title column documentation
```

Comment:
Super unimportant, but can you change this to be actually helpful docs? E.g., "The table containing all blog posts", "The post's unique ID", "The post's title"